website/docs/d/clusterScan.html.markdown

```diff
@@ -0,0 +1,53 @@
+---
+layout: "rancher2"
+page_title: "Rancher2: rancher2_cluster_scan"
+sidebar_current: "docs-rancher2-datasource-cluster_scan"
+description: |-
+  Get information on a Rancher v2 Cluster CIS Scan resource.
+---
+
+# rancher2\_cluster\_scan
+
+Use this data source to retrieve information about a Rancher v2 Cluster CIS Scan resource.
+
+## Example Usage
+
+```hcl
+data "rancher2_cluster_scan" "foo" {
+    cluster_id = <clusterID>
+    name = "foo"
+}
```

Comment:
Can we show an example of scheduled? I can't see wanting to run an adhoc scan with TF as the norm.

Reply:
This is a datasource intended just to get status about already run cluster scans.  cluster_scan resource didn't exist just by the reason you mention.

Reply:
Added examples on cluster and cluster template docs